### PR TITLE
Update UG: Quick start, addi, deletei

### LINKED ISSUE #20: Update User Guide (Jiacheng)
Update User Guide Sections: 
* Add inventory item
* delete inventory item

### PR DESCRIPTION
**Pull request by Jiacheng-y**
Updated following sections in UG : 
* Quick start
* Adding inventory item: `addi`
* Deleting inventory item: `deletei`

### CODE REVIEW

**Review by janelleljt — APPROVED** [review]
Looks good!

**Review by melissaharijanto — CHANGES_REQUESTED** [review]
`TrackO is a desktop app built for small business owners to help them **manage customer contacts** and orders...`

I think for this one we can change it to 'manage their inventory' instead? Because we are not dealing with customers anymore

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,36 +3,34 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+TrackO is a **desktop app built for small business owners to help them manage customer contacts and orders, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you prefer and are fast at typing, TrackO can get your order and inventory management tasks done faster than traditional GUI apps.
```
Comment (melissaharijanto) [inline docs/UserGuide.md]: The part I was talking about is here btw!

**Review by melissaharijanto — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (Jiacheng-y) [reply docs/UserGuide.md]: ah, good spot, thank you! I've made the changes in the new commit!

**Review by melissaharijanto — APPROVED** [review]
Looks good!

**Pull request merged**